src/Functions/Altinn.Auth.AuditLog.Functions/Models/AuthenticationEvent.cs

```diff
@@ -29,7 +29,7 @@ public class AuthenticationEvent
         /// <summary>
         /// Date, time of the authentication event. Set by producer of logevents
         /// </summary>
-        public DateTime Created { get; set; }
+        public DateTimeOffset Created { get; set; }
```

Comment:
This needs to be `DateTimeOffset?` - else you have no way of knowing if a value was provided.